### PR TITLE
refactor: generalise to organisations

### PR DESCRIPTION
**Pull request by mirdono**
Related to OP-3230

Where appropriate references to "administrative unit" in function names,
variables, parameters, etc. were replaced by "organization". In the process,
some documentation was added to most functions.
The actual renaming of the service will be done in a separate PR.

### CODE REVIEW

File: lib/queries.js [hunk 1]
```diff
@@ -7,48 +7,76 @@ import {
...
-export async function getAdministrativeUnitDetails(administrativeUnitdUuid) {
+
+/**
+ * Retrieve the necessary details for an organization from the triplestore.
+ * @typedef {object} OrganizationDetails
+ * @property {string} uri - the URI of the organization.
+ * @property {string} uuid - the UUID of the organization.
+ * @property {string} classification - the URI of the organization's
+ *     classification code.
+ * @property {string} [religionType] - If it is a worship administrative unit,
```
Comment (aliokan) [inline lib/queries.js]: **Suggestion**: 

```suggestion
 * @property {string|null} religionType - If it is a worship administrative unit,
```

File: lib/queries.js [hunk 2]
```diff
@@ -125,21 +171,27 @@ export async function createMandate(classification, governingBodiesInTime) {
...
 
-export async function createBestuursfunctie(
-  classification,
-  governingBodyInTime
-) {
+/**
+ * Create a new bestuursfunctie, with a given role, for the provided
+ * time-specialised governing body.
+ * @param {string} role - the URI of the role for the new bestuursfunctie.
+ * @param {string} governingBodyInTime ${2} - the URI of the governing body of
```
Comment (aliokan) [inline lib/queries.js]: **Question**: What is the purpose of `${2}`?

File: lib/queries.js [hunk 3]
```diff
@@ -58,64 +86,82 @@ export async function createGoverningBody(administrativeUnit, classification) {
...
+ * @param {string} [endDate] - the optional end data and time for the new
+ *     governing body.
+ * @returns {Promise<string>} the URI of the newly created time specialised
+ *     governing body.
+ */
 export async function createGoverningBodyInTime(
   governingBody,
   startDate,
-  endDate
+  endDate,
```
Comment (aliokan) [inline lib/queries.js]: ***Suggestion***: Remove the extra comma. I guess is coming from your code formatter. Also line 128, 164

**Review by aliokan — APPROVED** [review]
@mirdono I left some none blocker suggestions

File: lib/queries.js (hunk 2 above)
Comment (mirdono) [reply lib/queries.js]: No idea, it is something automatically added by my completions system, and  I have not gotten around to look further into it. I just forget to remove this one.

**Pull request merged**